site/en/blog/framework-tools-font-fallback/index.md

```diff
@@ -0,0 +1,402 @@
+
+
+#### Example
+
+Consider the following example: The desired web font is Lobster, falling back to Helvetica Neue and then Arial. In CSS, `font-family: Lobster, 'Helvetica Neue', Arial`.
+
+
+
+1. Import createFontStack from the core package: 
+
+    ```javascript
+        import { createFontStack } from '@capsizecss/core';
+    ```
+
+2. Import the font metrics for each of the desired fonts (see Font Metrics above): 
+    ```javascript
+        import lobster from '@capsizecss/metrics/lobster';
+        import helveticaNeue from '@capsizecss/metrics/helveticaNeue';
+        import arial from '@capsizecss/metrics/arial';`
+    ```
+
+
+3. Create your font stack, passing the metrics as an array, using the same order as you would via the font-family CSS property. 
+    ```javascript
+        const { fontFamily, fontFaces } = createFontStack([ 
+        lobster,
+        helveticaNeue,
+        arial,
+        ]);
+
+    ```
+
+
+This returns the following:
+
+
+```css
+    {
+       fontFamily: Lobster, 'Lobster Fallback: Helvetica Neue', 'Lobster Fallback: Arial',
+       fontFaces: [
+         {
+    	'@font-face' {
+          'font-family': '"Lobster Fallback: Helvetica Neue"';
+          src: local('Helvetica Neue');
+          'ascent-override': '115.1741%';
+          'descent-override': '28.7935%';
+          'size-adjust': '86.8251%';
+        }
+    	'@font-face' {
+          'font-family': '"Lobster Fallback: Arial"';
+          src: local('Arial');
+          'ascent-override': 113.5679%;
+          'descent-override': 28.392%;
+          'size-adjust': 88.053%;
+        }
+         }
+       ]
+    }
+```
+
+
+You must add the fontFamily and fontFaces code to your CSS. The following code shows how to implement it in a CSS style sheet, or within a `<style>` block.
+
+This will produce the following CSS:
+
+```css
+    <style type="text/css">
+    .heading {
+        font-family: {{ fontFamily }}
+    }
+
+    {{ fontFaces }}
+    </style>
+```
+
+You can also use the [@capsize/metrics](https://www.npmjs.com/package/@capsizecss/metrics) package to calculate the override values, and apply them to the CSS yourself.
+
+```javascript
+    const fontMetrics = require(`@capsizecss/metrics/inter`);
+    const fallbackFontMetrics = require(`@capsizecss/metrics/arial`);
+    const mainFontAvgWidth = fontMetrics.xAvgWidth / fontMetrics.unitsPerEm;
+    const fallbackFontAvgWidth = fallbackFontMetrics.xAvgWidth / fallbackFontMetrics.unitsPerEm;
+    let sizeAdjust = mainFontAvgWidth / fallbackFontAvgWidth;
+    let ascent = fontMetrics.ascent / (unitsPerEm * fontMetrics.sizeAdjust));
+    let descent = fontMetrics.descent / (unitsPerEm * fontMetrics.sizeAdjust));
+    let lineGap = fontMetrics.lineGap / (unitsPerEm * fontMetrics.sizeAdjust));
+```
+
+## Request for feedback
+
+We welcome feedback on your experiences using font metric overrides and `size-adjust` to improve fallback fonts.
```

Comment:
where should people leave this feedback?

Reply:
Removed this section.